src/index.ts

```diff
@@ -45,7 +46,13 @@ export const getKeypairFromFile = async (filepath?: string) => {
 export const getKeypairFromEnvironment = (variableName: string) => {
   const secretKeyString = process.env[variableName];
   if (!secretKeyString) {
-    throw new Error(`Please set '${variableName}' in environment.`);
+    // Generate a new keypair if one doesn't exist and add it to a `.env` file
```

Comment:
Having an easy way to save keypairs is good PR, but my concern with adding this to an existing function is that something called `getKeypairFromEnvironment()` shouldn't have any side effects. Generally things called `get()` should be fetching data, not changing it. Also I'm concerned if someone wants to get a keypair with a particular name, it doesn't exist, and they get it made - it's possible we could be silently throwing away errors. Maybe I have a keypair called `TEMP_WALLET` and run `getKeypairFromEnvironment('TEMPORARY_WALLET')` and what should have failed succeeds.

I'd called the code that appends keypairs something like `appendKeypairToFile()`. Whatever you think is the most obvious for users.